src/features/content-answers/ContentAnswersGridCard.tsx

```diff
@@ -0,0 +1,27 @@
+import React from 'react';
+
+import BoxAi from '../../icon/line/BoxAi';
+import Media from '../../components/media';
+
+import './ContentAnswersGridCard.scss';
+
+type Props = {
+    children: React.ReactNode;
+};
+
+const ContentAnswersGridCard = ({ children }: Props) => {
+    return (
+        <Media className="bdl-ContentAnswersGridCard" data-testid="content-answers-grid-card">
+            <Media.Figure>
+                <div className="bdl-BoxAIIcon-iconAvatar">
```

Comment:
[nit]
```suggestion
                <div className="bdl-BoxAiIcon">
```

Reply:
the icon is a descendant of `bdl-ContentAnswersGridCard`

`bdl-ContentAnswersGridCard-icon`